waku/handshake.go

```diff
@@ -6,12 +6,50 @@ import (
 	"io"
 	"math"
 	"reflect"
-	"strings"
+	"strconv"
 
 	"github.com/ethereum/go-ethereum/rlp"
 )
 
-var defaultMinPoW = math.Float64bits(0.001)
+type statusOptionKey uint
+
+type statusOptionKeyType uint
+
+type statusOptionKeyToType struct {
+	Idx  int
+	Key  statusOptionKey
+	Type statusOptionKeyType
+}
+
+const (
+	sOKTS statusOptionKeyType = iota // Status Option Key Type String
```

Comment:
```suggestion
	sOKTS statusOptionKeyType = iota + 1 // Status Option Key Type String
```

Avoid zero value as it's impossible to tell whether it's an uninitialized variable or actual value.

Reply:
Good point